grakn-graql-shell/src/main/resources/jetty-logging.properties

```diff
@@ -22,6 +22,6 @@
 
 # I'm not kidding, don't get rid of this please
 
-org.eclipse.jetty.util.log.class=org.eclise.jetty.util.log.StdErrLog
+org.eclipse.jetty.util.log.class=org.eclipse.jetty.util.log.StdErrLog
```

Comment:
Maybe this can die now? Unless my eye's deceive me it looks like you doing `1=1`